src/modules/demo_playback.rs

```diff
@@ -34,7 +34,9 @@ impl Module for DemoPlayback {
     }
 }
 
-static DEMOS: MainThreadRefCell<Vec<Vec<u8>>> = MainThreadRefCell::new(Vec::new());
+pub static DEMOS: MainThreadRefCell<Vec<Vec<u8>>> = MainThreadRefCell::new(Vec::new());
+pub static DEMOS_COUNT: MainThreadCell<usize> = MainThreadCell::new(0);
```

Comment:
Kinda seems like this should be in the place in capture which needs it and not here? Like, if you do bxt_cap_start_segment in the middle of playing demos, this will be wrong. Whereas in Capture you can set it at the point where it's first used

Reply:
Yea, it is just weird how I have pub state and stuffs, which is really weird because it is just me...... Gonna try moving things around and see how it looks.